packages/o2-blocks/babel.config.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+	extends: require.resolve( '@automattic/calypso-build/babel.config.js' ),
+	plugins: [
+		[
+			'@wordpress/import-jsx-pragma',
```

Comment:
This dependency seems to be missing now and the build fails. We'll need to figure out the right place to include it.

Reply:
```
npm run distclean && npm ci
npx lerna run prepare --scope="*/o2-blocks"
```

seem to wfm :thinking: 

But yeah, you're right, it's no longer in any `package.json`. I'll add it back to the root level `package.json`'s devDeps.

Reply:
> seem to wfm :thinking: 

:man_facepalming: Disregard, _someone_ was on `master` :roll_eyes:

Reply:
Also, I'm thinking of using the longform name (`@wordpress/babel-plugin-import-jsx-pragma`) here so people grepping that will find it (and won't be tempted to remove the dependency again).